modules/notification/notification.js

```diff
@@ -1,31 +1,21 @@
-/* electron */
-const rxIpc = require('rx-ipc-electron/lib/main').default;
-const log = require('electron-log');
-const Notification = require('electron').Notification;
+const rxIpc        = require('rx-ipc-electron/lib/main').default;
+const log          = require('electron-log');
+const path         = require('path');
+const Observable   = require('rxjs/Observable').Observable;
+const Notification = require('electron-native-notification');
 
-/* node */
-const path = require('path');
-
-const Observable = require('rxjs/Observable').Observable;
-
-/*
-    Register and IPC listener and execute notification.
-*/
 exports.init = function () {
-    rxIpc.registerListener('notification', function (title, desc, params) {
-        let notification = new Notification({
-            'title': title,
-            'body': desc,
-            'icon': path.join(__dirname, 'src/assets/icons/notification.png')
-        })
-        notification.show()
-        return Observable.create(observer => {
-            observer.complete(true);
-        });
-    });
+  rxIpc.registerListener('notification', (title, desc, params) => {
+    params = params ? params : { }; /* create param object if no param was given */
+    params.icon = '../../src/assets/icons/notification.png';
```

Comment:
relative path does not work in linux packaged version

Reply:
`src/assets/icons/notification.png` Is this work for linux ?